### ISSUE TITLE
build fail with autoreconf exit 1

### ISSUE THREAD

**Issue by axelrose** [strukturag/libde265#194]
Hello team,

I'm trying to build from latest git clone on macos.

The `./autogen.sh` failes reproducibly with
```
~/libde265 ❯❯❯ ./autogen.sh
autoreconf: Entering directory `.'
autoreconf: configure.ac: not using Gettext
autoreconf: running: aclocal --force -I m4
autoreconf: configure.ac: tracing
autoreconf: running: libtoolize --copy --force
libtoolize: putting auxiliary files in '.'.
libtoolize: copying file './ltmain.sh'
libtoolize: putting macros in AC_CONFIG_MACRO_DIRS, 'm4'.
libtoolize: copying file 'm4/libtool.m4'
libtoolize: copying file 'm4/ltoptions.m4'
libtoolize: copying file 'm4/ltsugar.m4'
libtoolize: copying file 'm4/ltversion.m4'
libtoolize: copying file 'm4/lt~obsolete.m4'
autoreconf: running: /usr/local/bin/autoconf --force
configure.ac:10: error: possibly undefined macro: AC_SUBST
      If this token and others are legitimate, please use m4_pattern_allow.
      See the Autoconf documentation.
configure.ac:136: error: possibly undefined macro: AC_DEFINE
autoreconf: /usr/local/bin/autoconf failed with exit status: 1
```

What could I do to continue?

My goal is to just have a libheif library to enable it in ImageMagick.

Tks, Axel.

**Comment by fancycode** [comment]
Have you installed `pkg-config`? This could cause issues if missing.

Which versions of autoconf, automake, libtool, m4 and pkg-config are you using?

**Comment by axelrose** [comment]
```
~/libde265 ❯❯❯ automake --version                                         
automake (GNU automake) 1.16

~/libde265 ❯❯❯ autoconf --version                                         
autoconf (GNU Autoconf) 2.68

~/libde265 ❯❯❯ which libtool      ## don't see how to check version
/usr/bin/libtool

~/libde265 ❯❯❯ m4 --version                                               
GNU M4 1.4.6

~/libde265 ❯❯❯ pkg-config --version                                       
0.25
```

**Comment by axelrose** [comment]
Solved by updating to pkg-config version 0.29.2 via `brew install pkg-config`
